hm_1/tasks/task_02_05.py

```diff
@@ -0,0 +1,25 @@
+def detect_index(some_iterable_object, position):
+    if isinstance(some_iterable_object, dict):
+        if position in list(some_iterable_object.keys()):
+            return list(some_iterable_object.keys()).index(position)
+        else:
+            return list(some_iterable_object.values()).index(position)
+    return some_iterable_object.index(position)
+
+
+def custom_range(some_iterable_object, start="", stop="", step=1):
```

Comment:
Can some_iterable_object have an empty string as a value?

Reply:
Yes, and it returns a "Wrong value"

Reply:
Or you ask about  this situation [''g','','o'] ?